src/librustdoc/html/highlight.rs

```diff
@@ -761,7 +807,7 @@ impl<'a> Classifier<'a> {
             TokenKind::CloseBracket => {
                 if self.in_attribute {
                     self.in_attribute = false;
-                    sink(Highlight::Token { text: "]", class: Some(Class::Attribute) });
+                    sink(Highlight::Token { text: "]", class: None });
```

Comment:
It's mostly to prevent running some code: if the `Class` is `None`, it returns early in `string_without_closing_tag`. But otherwise it doesn't have any impact on the output because it has the same class as the "parent" so it doesn't generate a `<span>`.